sw/device/tests/BUILD

```diff
@@ -2437,6 +2437,33 @@ opentitan_functest(
     ],
 )
 
+opentitan_functest(
+    name = "usbdev_logging_test",
+    srcs = ["usbdev_logging_test.c"],
+    cw310 = cw310_params(
+        timeout = "eternal",
```

Comment:
Eternal is a Bazel timeout that means - oddly - '1 hour' and the settings just comes from other usbdev tests that can be configured to run for that long. This test does require manual invocation and interaction on the host at present, so the timeout is rather irrelevant. In time it ought to be possible to run this test on FPGA with a physical host, at which point a lower timeout may be introduced.
I've removed the dv and Verilator targets for now, though, since there is no plan to introduce DPI host behavior to allow this test to pass.